UL.md

```diff
@@ -6,13 +6,16 @@ Develop a ubiquitous language for the domain of a slide show.
 
 | Concept | Sub concept | Meaning | Remarks |
 |---------|-------------|---------|---------|
-| slide show |  | an ordered sequence of slides |  |
+| slide show |  | a set of ordered sequences of the same slides |  |
```

Comment:
Misschien: A set of slides in combination with a set of sequences in which a subset of these slides can be shown?